apps/src/aichat/views/modelCustomization/PublishNotes.tsx

```diff
@@ -1,31 +1,51 @@
 import React from 'react';
+import {useSelector} from 'react-redux';
 
 import {StrongText} from '@cdo/apps/componentLibrary/typography/TypographyElements';
 import styles from '../model-customization-workspace.module.scss';
+import {LabState} from '@cdo/apps/lab2/lab2Redux';
+import {AichatLevelProperties, ModelCardInfo} from '@cdo/apps/aichat/types';
+import {DEFAULT_MODEL_CARD_INFO} from './constants';
+import {isHidden, isDisabled} from './utils';
 
-const INPUTS = [
-  {id: 'intended-uses', text: 'Intended Uses'},
-  {id: 'limitations-and-warnings', text: 'Limitations and Warnings'},
-  {id: 'testing-and-evaluation', text: 'Testing and Evaluation'},
+const INPUTS: {id: keyof ModelCardInfo; text: string}[] = [
```

Comment:
We can probably move this into constants and reuse between here and the levelbuilder edit view?

Reply:
Done!